frontend/src/components/common/player/player.tsx

```diff
@@ -0,0 +1,124 @@
+import AudioPlayer from 'react-h5-audio-player';
+import 'react-h5-audio-player/lib/styles.css';
+import styles from './styles.module.scss';
+
+type Props = {
+  src: string;
+  onClickPrevious?: () => void;
+  onClickNext?: () => void;
+};
+
+const MILLISECONDS_IN_SECOND = 1000;
+const SKIP_IN_MILLISECONDS = 15000;
+
+const Player: React.FC<Props> = ({ src, onClickPrevious, onClickNext }) => {
+  return (
+    <AudioPlayer
+      src={src}
+      showSkipControls={true}
+      progressJumpSteps={{
+        backward: SKIP_IN_MILLISECONDS,
+        forward: SKIP_IN_MILLISECONDS,
+      }}
+      customAdditionalControls={[]}
+      onClickNext={onClickNext}
+      onClickPrevious={onClickPrevious}
+      customIcons={{
+        play: (
+          <div
+            className={[
+              styles.playerCommonBtn,
+              styles.playerMainBtn,
+              styles.playerPlayBtn,
+            ].join(' ')}
```

Comment:
```suggestion
            className={[
              styles.playerCommonBtn,
              styles.playerMainBtn,
              styles.playerPlayBtn,
            ].join(' ')}
```
use `getValidClasses` helper